content/en/tracing/compatibility_requirements/java.md

```diff
@@ -87,6 +87,8 @@ Don't see your desired web frameworks? Datadog is continually adding additional
 | Rabbit AMQP              | 2.7+        | Fully Supported | `amqp`, `rabbitmq`                             |
 | Spring WebClient         | 5.0+        | Fully Supported | `spring-webflux`, `spring-webflux-client`      |
 
+**Note**: Our Kafka Integration works with version `0.11+` as this version of Kafka supports the Header API, where we inject/extract trace context. If you a running a mixed version environment, the Kafka broker will sometimes report the newer version of Kafka. This can cause an issue as the tracer will try to inject headers when it’s not supported by the local producer, or older consumers will be unable to consume the message because of the presence of headers. To prevent this issue if you are running a mixed version Kafka environment with versions older than 0.11, context propagation can be disabled with the following environment variable: `DD_KAFKA_CLIENT_PROPAGATION_ENABLED=false`.
```

Comment:
"If you **are** running"

Reply:
"sometimes **incorrectly** report the newer version of Kafka"

Reply:
👍